tests/unittests/test_currently_syncing.py

```diff
@@ -0,0 +1,109 @@
+import unittest
+from unittest import mock
+from tap_github.sync import (update_currently_syncing_repo, update_currently_syncing,
+                             get_ordered_stream_list, get_ordered_repos)
+
+class TestGetOrderedStreamList(unittest.TestCase):
+    """
+    Test `get_ordered_stream_list` function to get ordered list od streams
+    """
+
+    def test_for_interrupted_sync(self):
+        """Test if sync was interrupted"""
+        expected_list = ['releases', 'review_comments', 'reviews', 'stargazers', 'team_members',
+                         'team_memberships', 'teams', 'assignees', 'collaborators', 'comments',
+                         'commit_comments', 'commits', 'events', 'issue_events', 'issue_labels',
+                         'issue_milestones', 'issues', 'pr_commits', 'project_cards',
+                         'project_columns', 'projects', 'pull_requests']
+        final_list = get_ordered_stream_list("releases")
+
+        # Verify with expected ordered list of streams
+        self.assertEqual(final_list, expected_list)
+
+    def test_for_completed_sync(self):
+        """Test if sync was not interrupted"""
+        expected_list = ['assignees', 'collaborators', 'comments', 'commit_comments',
+                         'commits', 'events', 'issue_events', 'issue_labels', 'issue_milestones',
+                         'issues', 'pr_commits', 'project_cards', 'project_columns', 'projects',
+                         'pull_requests', 'releases', 'review_comments', 'reviews', 'stargazers',
+                         'team_members', 'team_memberships', 'teams']
+        final_list = get_ordered_stream_list(None)
+
+        # Verify with expected ordered list of streams
+        self.assertEqual(final_list, expected_list)
+
+class TestGetOrderedRepos(unittest.TestCase):
+
+    """
+    Test `get_ordered_repos` function to get ordered list repositories
+    """
+    repo_list = ["org/repo1", "org/repo2", "org/repo3", "org/repo4", "org/repo5"]
+
+    def test_for_interupted_sync(self):
+        """Test if sync was interrupted"""
```

Comment:
Update test comment here too as mentioned in the last comment.

Reply:
Added detailed comment